singularity-eos/eos/modifiers/ramps_eos.hpp

```diff
@@ -117,7 +117,7 @@ class BilinearRampEOS : public EosBase<BilinearRampEOS<T>> {
 
   PORTABLE_INLINE_FUNCTION
   Real get_ramp_dpdrho(Real rho) const {
-    const Real dpdr{rho < r0_ ? 0.0 : rho < rmid_ ? ratio(a_, r0_) : ratio(b_, r0_)};
+    const Real dpdr{rho < rmid_ ? ratio(a_, r0_) : ratio(b_, r0_)};
```

Comment:
Consider removing the branching here if possible by using masks instead?

Reply:
I think we'll defer this since it might make sense to replace these all wholesale in the future.

Reply:
see https://github.com/lanl/singularity-eos/issues/319